sdk/storage/Azure.Storage.Blobs/tests/BlockBlobClientTests.cs

```diff
@@ -348,9 +351,12 @@ public async Task StageBlockFromUriAsync_Min()
             BlockBlobClient destBlob = InstrumentClient(test.Container.GetBlockBlobClient(GetNewBlobName()));
 
             // Act
-            await destBlob.StageBlockFromUriAsync(sourceBlob.Uri, ToBase64(GetNewBlockName()));
+            await RetryAsync(
+                async () => await destBlob.StageBlockFromUriAsync(sourceBlob.Uri, ToBase64(GetNewBlockName())),
```

Comment:
Any possible issue with calling`GetNewBlockName` multiple times on retry?

Reply:
I don't think so, but there isn't a good reason for us to generate a new name on each retry.